Sources/ForageSDK/Foundation/Network/Collector.swift

```diff
@@ -12,10 +12,11 @@ import BasisTheoryElements
 let tokenDelimiter = ","
 let tokenKey = "card_number_token"
 
-public protocol VaultCollector {
+internal protocol VaultCollector {
```

Comment:
not sure why this was public before?

@dleis612

Reply:
uhhhh definitely want this to be internal.